sao/src/theme/coog/sao.less

```diff
@@ -39,8 +39,8 @@
 // Simple fields readonly styles
 .coog-readonly-field-mixin {
     background-color:     @color-lightest-grey;
-    border-top-left-radius:   5px;
-    border-top-right-radius:  5px;
+    border-top-left-radius:   5px !important;
+    border-top-right-radius:  5px !important;
```

Comment:
not nice

Reply:
certes, mais il y a du standard dans Tryton qui passe avant. En général j'essaye d'éviter :(